src/index.ts

```diff
@@ -60,4 +60,5 @@ export {
   RENDERMODE_PATHTRACE,
   RENDERMODE_RAYMARCH,
   SKY_LIGHT,
+  type CameraTransform,
```

Comment:
This throws a babel warning, any idea why it might be triggering off of this and not the similar declarations on lines 47-49?

Reply:
I'm not seeing a similar error, and don't have any great theories on why you would be

Reply:
> This throws a babel warning, any idea why it might be triggering off of this and not the similar declarations on lines 47-49?

maybe because you didn't use `import type` to import it?

Reply:
That was it, thank you!